### ISSUE TITLE
Fix Hubspot robot test's secrets value in CI

### ISSUE THREAD

**Issue by cmin764** [robocorp/rpaframework#740]
CI has [failing](https://github.com/robocorp/rpaframework/actions/runs/3688305681) robot tests due to `401 Unauthorized` given the either outdated or missing `HUBSPOT_TESTVARS` secret value.

<img width="652" alt="Screenshot 2022-12-14 at 12 24 57" src="https://user-images.githubusercontent.com/709053/207575064-c0d62160-1efd-4efd-a82f-729f7413ba92.png">

### ToDo
- [ ] Update the secret in GitHub so it unblocks the flow in order to deploy docs and merge PRs nicely. (put some instance credentials in 1Password in case somebody else needs to generate new API keys)
- [x] **In order for this to not happen again, let's add a failback check and if the credentials are not good, the tests will be just skipped or kinda xfailed.**
- [x] Maybe it looks more RFish to have a _HubspotTestvars.py_ instead of _hubspot_testvars.py_.
  - [x] Ensure such template script already exists and given the fake creds inside, it will fail the login, thus skipping the tests. (rather than relying on the absence/existence of such file to decide if we skip or not the tests)
- [ ] Optional (if needed): Create a helper script inside of _main/tests/scripts_ which will take the content of an existing _HubspotTestvars.py_ file and output the content that should be placed in the GH secret value. (like base64 encoding if that's not already done by GitHub)

Internal Slack [thread](https://robocorptechnologies.slack.com/archives/CS71JM9T7/p1671013668245459)
